src/libs/ajax.js

```diff
@@ -51,11 +52,28 @@ const withCancellation = wrappedFetch => async (...args) => {
   }
 }
 
+//const captureRetryFailure = (...args) => _.once(() => {
+//      console.log('capture failure')
+//      const errorAddress = _.flow(
+//        _.values,
+//        _.find(v => _.includes(v, args[0]) && v)
+//      )([..._.values(getConfig()), 'https://storage.googleapis.com/'])
+//      console.log("errorAddress", errorAddress)
+//      Ajax().Metrics.captureEvent(Events.requestFailed, { test: errorAddress })
+//  })
+
 const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
   const timeout = 5000
   const somePointInTheFuture = Date.now() + timeout
   const maxDelayIncrement = 1500
   const minDelay = 500
+  const captureFailure = _.once(() => {
+    const errorAddress = _.flow(
+      _.values,
+      _.find(v => _.includes(v, args[0]) && v)
+    )([..._.values(getConfig()), 'https://storage.googleapis.com/'])
+    Ajax().Metrics.captureEvent(Events.requestFailed, { test: errorAddress })
```

Comment:
Is there a more descriptive name for this property than `test`?

Reply:
Not too creative myself with names, have any suggestion I could use?

Reply:
Depending on the answer to this question: https://github.com/DataBiosphere/terra-ui/pull/3072#discussion_r880956884, maybe something like `apiUrl` or `requestUrl`?